cypress/support/commands.js

```diff
@@ -391,7 +391,7 @@ Cypress.Commands.add('createUser', user => {
 		cy.get('form[id=userRoleForm]').contains(role).click();
 	});
 	cy.get('form[id=userRoleForm] button[id^=submitFormButton]').click();
-	cy.get('span[id$="-username"]:contains("' + Cypress.$.escapeSelector(user.username) + '")');
+        cy.waitJQuery();
```

Comment:
It looks like this actually changes the check, so that it's no longer waiting to see if the user was actually created and appears in the list. Was that intentional?

Reply:
I think the problem was that this check assumed that the user was going to appear on the current page, and that may not be the case. It showed up when I temporarily removed the ordering code -- the ordering has been reinstituted, but I still think we can't assume the new user will be on the current page (think e.g. of adding an 11th user on a 10-user list).